src/cli.rs

```diff
@@ -47,21 +51,37 @@ pub struct Args {
     reward_address: Option<String>,
 }
 
-#[derive(Debug, Clone, ValueEnum)]
+impl Args {
+    pub fn init() -> Result<Config, AppError> {
+        let matches = Self::command().get_matches();
+        let arguments = Self::with_layers(&[
+            Layer::Clap(matches),
+            Layer::Env(Some(String::from("BLOCKFROST_"))),
```

Comment:
If you move `Layer::Env` before (above) `Layer::Clap`, won’t it favor CLI arguments? I don’t know this library, but intuitively it could work like that

Reply:
yes, you are right. I did the exact implementation and in the meantime, I was trying to think of any reason why this would make sense. Now this issue is fixed